pkg/occlient/occlient.go

```diff
@@ -2520,21 +2520,14 @@ func (c *Client) GetOnePodFromSelector(selector string) (*corev1.Pod, error) {
 // During copying local source components, localPath represent base directory path whereas copyFiles is empty
 // During `odo watch`, localPath represent base directory path whereas copyFiles contains list of changed Files
 func (c *Client) CopyFile(localPath string, targetPodName string, targetPath string, copyFiles []string) error {
-	isSingleFileTransfer := isSingleFileTransfer(copyFiles)
```

Comment:
Now that `isSingleFileTransfer` is no longer being used, please remove it as well

Reply:
Sure
I am verifying the applicability of https://github.com/redhat-developer/odo/issues/950 with this PR

Reply:
Ok I think it works

Reply:
> Now that `isSingleFileTransfer` is no longer being used, please remove it as well

@geoand Done